server/src/main/java/org/elasticsearch/index/engine/NoopEngine.java

```diff
@@ -98,6 +98,9 @@ public void close() {
 
             // The translog is opened and closed to validate that the translog UUID from lucene is the same as the one in the translog
             Translog translog = openTranslog(engineConfig, translogDeletionPolicy, engineConfig.getGlobalCheckpointSupplier());
```

Comment:
wrap this in try finally?

Reply:
Yes good catch, I thought about that last night but forgot to do it, I'll update that